ruby/ruby/IceGrid.rb

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) ZeroC, Inc. All rights reserved.
+#
+
+require 'IceGrid/Admin.rb'
+require 'IceGrid/Descriptor.rb'
+require 'IceGrid/FileParser.rb'
+require 'IceGrid/Registry.rb'
+require 'IceGrid/UserAccountMapper.rb'
```

Comment:
We should include all generated files. `Exception.rb` and `Session.rb` are missing.